src/services/userServices.js

```diff
@@ -2,10 +2,10 @@ import model from '../models';
 
 const { User } = model;
 const users = {
-
   getByKey: key => User.findOne({
     where: key,
     logging: false
-  }),
+  })
 };
+
 export default users;
```

Comment:
This file does not follow team convention, we now have it as `./src/modules/users/users.services.js` try remove it